Sources/StreamVideo/WebRTC/WebRTCClient.swift

```diff
@@ -120,10 +120,23 @@ class WebRTCClient: NSObject {
     private(set) var callSettings = CallSettings()
     private(set) var videoOptions = VideoOptions()
     private let environment: WebSocketClient.Environment
+    private let apiKey: String
     
+    private var migratingSignalService: Stream_Video_Sfu_Signal_SignalServer?
+    private var migratingWSClient: WebSocketClient?
+    private var migratingToken: String?
+    private var fromSfuName: String?
+    private var temp: PeerConnection?
```

Comment:
Can we make this `temporarySubscriber` or `migrationTemporaryPeerConnection`?

Reply:
done